TTHAnalysis/python/plotter/ttH-multilepton/ttH_plots.py

```diff
@@ -315,6 +320,11 @@ def doprescale3l(x,torun):
         x = base('2lss')
         x = x.replace('mca-2lss-mc.txt','mca-2lss-mcdata-ttbar.txt')
         x = x.replace('--maxRatioRange 0 3','--maxRatioRange 0.8 1.2')
+        x = x.replace('--rebin 4','')
+        if '_appl' in torun:
+            x = add(x,"-X TT")
```

Comment:
more prudent to add ^ in front, as for many other exclude/invert/enable commands